scratch_launch-docker.sh

```diff
@@ -0,0 +1 @@
+docker run --name postgres -e POSTGRES_PASSWORD=postgres -d -p 5432:5432  --mount type=bind,source=$(realpath tests),target=/app postgres
```

Comment:
This file can be removed. Better is to add the text in a README.md in a section describing how to launch a postgresql server. Also, the mount-option is rather cumbersome. For an example postgresql-server running in docker you may create a docker volume or just point to some persistent data storage location.
```

docker volume create pgdata
docker run  -d --name postgres \
                   -e POSTGRES_PASSWORD=postgres \
                   -p 5432:5432 \
                   -v pgdata:/var/lib/postgresql/data \
                   --restart always postgres
```